### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## What type of PR is this?

- cleanup

## What this PR does / why we need it:

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/urfave-cli/commit/02977c8eef931bf0684a3b12d568f834f7c5b263#commitcomment-87654132

## Special notes for your reviewer:

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

## Testing

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/urfave-cli/commit/d23e518570f7769a29d0b461f1662342b9d669d5

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/v1-bug-report.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ _**( Put the version of urfave/cli that you are using here )**_
 
 - [ ] Are you running the latest v1 release? The list of releases is [here](https://github.com/urfave/cli/releases).
 - [ ] Did you check the manual for your release? The v1 manual is [here](https://github.com/urfave/cli/blob/main/docs/v1/manual.md).
-- [ ] Did you perform a search about this problem? Here's the [Github guide](https://help.github.com/en/github/managing-your-work-on-github/using-search-to-filter-issues-and-pull-requests) about searching.
+- [ ] Did you perform a search about this problem? Here's the [GitHub guide](https://help.github.com/en/github/managing-your-work-on-github/using-search-to-filter-issues-and-pull-requests) about searching.
```
Comment (jsoref) [inline .github/ISSUE_TEMPLATE/v1-bug-report.md:18]: brand

File: .github/ISSUE_TEMPLATE/v1-bug-report.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ link a failing build here.
 ## Want to fix this yourself?
 
 We'd love to have more contributors on this project! If the fix for
-this bug is easily explained and very small, free free to create a
+this bug is easily explained and very small, feel free to create a
```
Comment (jsoref) [inline .github/ISSUE_TEMPLATE/v1-bug-report.md:62]: atypical correction

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine

**Review by abitrolly — APPROVED** [review]
LGTM.

**Comment by abitrolly** [comment]
I would probably squash it into one commit.

**Comment by jsoref** [comment]
I'm happy to squash or let you squash-merge if you like.

**Comment by dearchap** [comment]
@jsoref @abitrolly I've enabled squash commit for this repo. Go ahead and squash merge

**Comment by abitrolly** [comment]
Done. Thanks. :D

**Pull request merged**